express-api/src/controllers/ltsa/ltsaController.ts

```diff
@@ -20,7 +15,10 @@ export const getLTSA = async (req: Request, res: Response) => {
             "bearerAuth": []
       }]
    */
+  const pid = req.query.pid as string;
+  const getLandTitleInfo = await ltsaService.processLTSARequest(pid);
 
-  // TODO: Replace stub response with controller logic
-  return stubResponse(res);
+  if (!getLandTitleInfo) {
```

Comment:
I think we might never trigger this. The service throws errors if it fails. It doesn't just return a response with no data.

Reply:
Implemented suggested change